### ISSUE TITLE
Remove legacyExplorer for loopback 3.0

### ISSUE THREAD

**Issue by davidcheung** [strongloop/loopback#2037]
Are there only references to the legacyExplorer? 

Related to #2009 
See commit: https://github.com/strongloop/loopback/commit/1818a8fb345c127d30f59b28f61590605efe8567
https://github.com/strongloop/loopback-workspace/pull/191
https://github.com/strongloop/loopback/search?q=legacyExplorer&type=Code&utf8=%E2%9C%93

Task: 
- [x] Remove legacyExplorer 

>~~https://docs.strongloop.com/display/public/LB/config.json#config.json-Top-levelproperties table should state that the default is true.~~ no longer relevant

**Comment by gunjpan** [comment]
This task also needs updating templates,in `loopback-workspace`, used for scaffolding loopback application

**Comment by gunjpan** [comment]
@bajtos: Should there be a separate issue for it? or want to cover under this _(in that case, the PR to loopback needs to be merged first)_ , thoughts?

**Comment by bajtos** [comment]
@gunjpan good point, I just wanted to point loopback-workspace out :) Please implement the change as part of this user story.

**Comment by bajtos** [comment]
@gunjpan both pull requests were landed, I am closing this story as done. Feel free to reopen if there is still some work remaining.
